src/git.rs

```diff
@@ -143,6 +153,20 @@ impl Repository {
                 )
                 .parse()?,
             ),
+            Repository::Forgejo {
+                server,
+                owner,
+                repo,
+            } => Some(
+                format!(
+                    "{server}{owner}/{repo}/archive/{revision}.tar.gz",
+                    server = server,
+                    owner = owner,
+                    repo = repo,
+                    revision = revision,
```

Comment:
```suggestion
                    "{server}{owner}/{repo}/archive/{revision}.tar.gz",
```

Those shouldn't be necessary since a couple of Rust versions